torch/onnx/utils.py

```diff
@@ -403,7 +404,9 @@ def _run_symbolic_function(g, n, inputs, env, aten=False):
             return None
 
         elif ns == "aten":
-            if aten:
+            is_exportable_aten_op = hasattr(torch.onnx.symbolic, op_name)
+            if (operator_export_type == OperatorExportTypes.ONNX_ATEN) or \
+                    (not is_exportable_aten_op and operator_export_type == OperatorExportTypes.ONNX_ATEN_FALLBACK):
```

Comment:
Can you please split the two branches in this if into variables and use them in here? It's very hard to read at the moment.